### ISSUE TITLE
Use `classnames` library for complex class names

### ISSUE THREAD

**Issue by Ishirak** [versionpress/versionpress#1089]
This issue is part of a frontend code review, see #1083.

JSX attribute **className** can sometimes contain a pretty long string, which depends on actual state, properties or context of a React.js component.

In the front end codebase, finding such a string can be sometimes a little bit unclear. But React.js community is a large one, so it already possess a utility [classnames](https://github.com/JedWatson/classnames), which makes finding **className** string a breeze.

For example,
```JSX
const className = 'details-row' + (commit.isEnabled ? '' : 'disabled') + (this.props.loading ? ' loading' : '');
```
can be converted into
```JSX
const className = classNames({
    'details-row': true,
    'disabled': !commit.isEnabled,
    'loading': this.props.loading
});
```
which is much cleaner and readable.

**Comment by vasek17** [comment]
👍 [classnames](https://github.com/JedWatson/classnames) for complex class names

**Comment by borekb** [comment]
Agree, let's do it. (I've also updated the title, it was not very clear to me before.)

**Comment by borekb** [comment]
:ship: Shipped in [4.0-alpha1](https://github.com/versionpress/versionpress/releases/tag/4.0-alpha1).
